project_2/include/link.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
```

Comment:
If we with this PR to close issues https://github.com/Cloud-Storage-Team/algnet/issues/30 and https://github.com/Cloud-Storage-Team/algnet/issues/31 then the code should be moved to the root folder.